File: Java/백준/Bronze/8321. Tables/Tables.java
import java.util.*;
import java.io.*;

public class Main {

    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    static StringTokenizer st;

    public static void main(String[] args) throws IOException {

        int n, k, s, target, sum = 0, cnt = 0;
        st = new StringTokenizer(br.readLine());
        n = Integer.parseInt(st.nextToken());
        k = Integer.parseInt(st.nextToken());
        s = Integer.parseInt(st.nextToken());

        Integer[] arr = new Integer[n];
        st = new StringTokenizer(br.readLine());
        for (int i = 0; i < n; i++) {
            arr[i] = Integer.parseInt(st.nextToken());
        }

        Arrays.sort(arr, (a, b) -> b - a);
        target = k * s;
        for (int num : arr) {
            sum += num;
            cnt++;
            if (target <= sum)
                break;
        }

        System.out.println(cnt);
    }
}

File: Python/백준/Bronze/8321. Tables/Tables.py
from sys import stdin

input = stdin.readline

n, k, s = map(int, input().split())
lst = sorted(map(int, input().split()), reverse=True)

sum = 0
cnt = 0
target = k * s
for i in range(n):
    sum += lst[i]
    if sum >= target:
        cnt = i + 1
        break

print(cnt)
